cli/cfncluster/cfnconfig.py

```diff
@@ -23,15 +23,25 @@
 import json
 import urllib.request, urllib.error, urllib.parse
 from . import config_sanity
-import boto.cloudformation
+import boto3
+from botocore.exceptions import ClientError
 
 def getStackTemplate(region, aws_access_key_id, aws_secret_access_key, stack):
-
-    cfn_conn = boto.cloudformation.connect_to_region(region,aws_access_key_id=aws_access_key_id,
-                                                 aws_secret_access_key=aws_secret_access_key)
+    cfn = boto3.client('cloudformation', region_name=region,
+                       aws_access_key_id=aws_access_key_id,
+                       aws_secret_access_key=aws_secret_access_key)
     __stack_name = ('cfncluster-' + stack)
-    __stack = cfn_conn.describe_stacks(stack_name_or_id=__stack_name)[0]
-    __cli_template = [p.value for p in __stack.parameters if p.key == 'CLITemplate'][0]
+
+    try:
+        __stack = cfn.describe_stacks(StackName=__stack_name).get('Stacks')[0]
+    except ClientError as e:
+        if e.response.get('Error').get('Message').endswith('does not exist'):
```

Comment:
Could we use the same error management pattern used for cfncluster.py., i.e. log the ClientError message and exit?

Reply:
we can't log it, but we can print and cleanly exit.